.github/workflows/testing.yml

```diff
@@ -467,37 +467,93 @@ jobs:
-
-       - name: Set up Go
-         uses: actions/setup-go@v2
-         with:
-           go-version: 1.17
-
-       - name: Make Coverage Directory
-         working-directory: ./
-         run: mkdir coverage
-
-       - name: Build API.go
-         working-directory: ./api
-         run: go build
-
-       - name: Start API Server
-         working-directory: ./api
-         run: go run api.go &
-
-       - name: Install python dependencies
-         working-directory: ./
-         run: |
-           python3 -m pip install build pytest
-           python3 -m build ./client/
-           python3 -m pip install client/dist/*
-
-       - name: Testing
-         env:
-           API_ADDRESS: "localhost:7878"
-         working-directory: ./client/tests
-         run: pytest
+      - name: Download Working Compiled Directories
+        uses: actions/download-artifact@v3
+        with:
+          name: compiled-workdir
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Make Coverage Directory
+        working-directory: ./
+        run: mkdir coverage
+
+      - name: Build API.go
+        working-directory: ./api
+        run: go build
+
+      - name: Start API Server
+        working-directory: ./api
+        run: go run api.go &
+
+      - name: Install python dependencies
+        working-directory: ./
+        run: |
+          python3 -m pip install build pytest
+          python3 -m build ./client/
+          python3 -m pip install client/dist/*
+
+      - name: Testing
+        env:
+          API_ADDRESS: "localhost:7878"
+        working-directory: ./client/tests
+        run: pytest
+  schedule:
+    runs-on: ubuntu-latest
+    name: Schedule Jobs
+    environment: Integration testing
+    needs: setup
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install grpc_tools
+        run: pip install grpcio-tools
+
+      - name: Install Protobuf
+        run: sudo snap install protobuf --classic
+
+      - name: Setup Proto
+        run: ./gen_grpc.sh
+      - name: Start minikube
+        uses: medyagh/setup-minikube@master
+      - name: build docker images and add to minikube
+        run: |
+          eval $(minikube docker-env)
+          docker build . --file runner/Dockerfile --tag local/worker:stable
+          docker build . --file coordinator/scheduletest/Dockerfile --tag local/scheduletester:stable
+          docker build . --file metadata/Dockerfile --tag local/metadata:stable
+          minikube image ls
+      - name: Set up Helm
+        run: |
+          echo -n "verifying images:"
+          docker images
+          helm repo add jetstack https://charts.jetstack.io
+          helm repo update
+          helm install certmgr jetstack/cert-manager --set installCRDs=true --version v1.8.0 --namespace cert-manager --create-namespace
+          helm install featureform ./charts/featureform --set global.repo=local --set global.pullPolicy=Never
```

Comment:
If you dont need the full stack you can just run ./charts/featureform/charts/metadata to reduce the load. This also makes it so you dont have to install the cert manager or ingresses